### PR TITLE
Fix implicit nullable parameters

### PR DESCRIPTION
**Pull request by KerianMM**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no
| License       | MIT

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
Can you please double check older branches? We should apply this to the oldest branches where it applies.

File: src/Symfony/Bridge/PsrHttpMessage/Tests/Fixtures/ServerRequest.php [hunk 1]
```diff
@@ -26,10 +26,10 @@ class ServerRequest extends Message implements ServerRequestInterface
     public function __construct(
         string $version = '1.1',
         array $headers = [],
-        StreamInterface $body = null,
+        StreamInterface|null $body = null,
```
Comment (xabbuh) [inline src/Symfony/Bridge/PsrHttpMessage/Tests/Fixtures/ServerRequest.php]:
```suggestion
        ?StreamInterface $body = null,
```

**Comment by KerianMM** [comment]
> Can you please double check older branches? We should apply this to the oldest branches where it applies.

Done on 6.4 (I had a doubt, no need to base on older branch ? 6.4 is ok ?), I'll make another PR on 7.0 for TypeInfo.

**Comment by stof** [comment]
We should start such fixes in the 5.4 branch as that's the oldest maintained branch.

**Pull request closed without merging**